### PR TITLE
Add Soviet11b

### PR DESCRIPTION
**Pull request by Smittytron**
Functionally the same code as in  #19158

### CODE REVIEW

**Comment by darcyrush** [comment]
Plays fine on hard

File: mods/ra/maps/soviet-11b/soviet11b-AI.lua [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+PatrolD = { Rocket7, Rocket8 }
+PatrolPathA = { RocketPath1.Location, RocketPath2.Location }
+PatrolPathB = { RocketPath3.Location, RocketPath4.Location }
+PatrolPathC = { RocketPath5.Location, RocketPath6.Location }
+PatrolPathD = { RocketPath7.Location, RocketPath8.Location }
+
+Helipads = { Helipad1, Helipad2, Helipad3, Helipad4 }
+HeliType = { "heli" }
+
+DestoryerSquad = { "dd", "dd" }
```
Comment (reaperrr) [inline mods/ra/maps/soviet-11b/soviet11b-AI.lua]: My comment from 11a applies here, too.

File: mods/ra/maps/soviet-11b/soviet11b-AI.lua [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+{
+	easy = 6,
+	normal = 5,
+	hard = 4
+}
+
+AlliedShips =
+{ 
+	easy = { "pt", "pt", "dd" },
+	normal = { "dd", "pt" },
```
Comment (reaperrr) [inline mods/ra/maps/soviet-11b/soviet11b-AI.lua]: My comment from 11a applies here, too.

**Comment by Smittytron** [comment]
Updated

**Comment by Smittytron** [comment]
Rebased

**Review by reaperrr — APPROVED** [review]
Didn't notice any issues while playing, LGTM

**Comment by reaperrr** [comment]
Didn't notice any issues while playing, lgtmhttps://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/85416c15b4120079e2c28938203809c7658082dc...d0a9932c5ded6e0e275834657835f1993a31061d

**Pull request merged**